airflow-core/src/airflow/api_fastapi/execution_api/routes/task_instances.py

```diff
@@ -130,7 +130,8 @@ def ti_run(
 
     # We exclude_unset to avoid updating fields that are not set in the payload
     data = ti_run_payload.model_dump(exclude_unset=True)
-
+    if ti.next_kwargs:
+        data.pop("start_date")
```

Comment:
Could you add a test too please

Reply:
yes, just wanted to see if maybe one was already there

Reply:
that i could just change

Reply:
test added